src/test/java/com/example/pfairplayservice/jpa/repository/MemberRepositoryIntegrationTest.java

```diff
@@ -28,6 +30,12 @@
     @Autowired
     private MemberRepository memberRepository;
 
+    @Autowired
+    private TeamRepository teamRepository;
+
+    @Autowired
+    private MemberTeamRepository memberTeamRepository;
+
```

Comment:
This class is `MemberRepositoryIntegrationTest`.
Create other classes for TeamRepository and MemberTeamRepository.

Reply:
ditto